ports/servoshell/minibrowser.rs

```diff
@@ -1,7 +1,3 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
All the comments seem to be deleted. This is probably because your ide formatted the file on save, so maybe try to take a look into the settings to preserve comments, and then you can add them back. There's also very helpful documentation [here](https://github.com/servo/servo/blob/main/docs/HACKING_QUICKSTART.md#editor-support) to set up your editor of choice.

Reply:
thanks for the input. I resolved that.